src/EventStore.Projections.Core/Services/Processing/EmittedStream.cs

```diff
@@ -269,6 +269,10 @@ public class StreamMetadata {
 				return;
 			if (message.CorrelationId != _pendingRequestCorrelationId)
 				return;
+			if (message.Result == ReadStreamResult.StreamDeleted) {
+				throw new InvalidOperationException($"Stream : {_streamId} is deleted. Cannot emit events to it");
```

Comment:
The error message looks much better, but the projection is no longer stopping because of it.
This can be dangerous because while the projection will continue projecting to other streams, it will no longer emit events to the deleted stream, and the user may never be notified of this.

If you call `Failed` here like the other errors below, the projection will correctly report the error and stop:

```suggestion
				Failed($"Stream : {_streamId} is deleted. Cannot emit events to it");
				return;
```

Reply:
Makes sense. I have updated it.